### ISSUE TITLE
Authentication for microservices

### ISSUE THREAD

**Issue by mars3142** [marcushellberg/hilla-microservices#1]
How would you implement JWT authentication for the microservices and where can the bff get the JWT to send it, if it's using the default oauth implementation of spring boot?

Could you add additional code for just one microservice?

**Comment by marcushellberg** [comment]
Yeah. Essentially you'd make the Hilla app the oauth client that performs the auth and then pass the JWT to the other services that would act as resource servers. I'll see if I can find time to add auth to the example

**Comment by hezhezhez** [comment]
Hello Marcus,
Did you get time to implement JWT auth example for microservices?
